Packs/AzureActiveDirectory/Integrations/AzureADIdentityProtection/AzureADIdentityProtection.yml

```diff
@@ -425,9 +456,137 @@ script:
       the targeted user's risk level to none.
     execution: false
     name: azure-ad-identity-protection-risky-user-dismiss
+  - arguments:
+    - default: false
+      description: The date and time that the risky user was last updated, formatted as `<number> <time unit>`, e.g., `12 hours`, `7 days`.
```

Comment:
```suggestion
      description: The date and time since the risky user was last updated, formatted as `<number> <time unit>`, e.g., `12 hours`, `7 days`.
```